frontend/src/core/helpers.js

```diff
@@ -132,18 +132,18 @@ define(function(require){
       return new Handlebars.SafeString(html);
     },
 
-    pickCSV: function (list, key, separator) {
-      var vals = [];
-      separator = (separator && separator.length) ? separator : ',';
-      if (list && list.length) {
-        return vals.join(separator);
+    pickCSV: function(list, key, separator) {
```

Comment:
Could we get some dock blocks on these I'm struggling to understand what the goal of this function is.

Reply:
Most of the functions in this file are before our time